openedx_tagging/core/tagging/models/base.py

```diff
@@ -119,7 +121,6 @@ class Taxonomy(models.Model):
         ),
     )
     description = MultiCollationTextField(
-        null=True,
```

Comment:
Django convention is not not use NULL for text fields, unless there's a very good reason, which doesn't seem to be the case here.